src/components/property-detail/Gallery.js

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+const SubImages = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const Gallery = (props) => {
+  const mainImageUrl = props.images[0].templatedUrl.replace('{size}', '800x480');
+  const imageUrlLeft = props.images[1].templatedUrl.replace('{size}', '264x128');
```

Comment:
please rename it to `leftImageUrl`, same as `middleImageUrl` and `rightImageUrl`